src/home_robot/home_robot/manipulation/grasping.py

```diff
@@ -60,7 +60,7 @@ def plan_to_grasp(
 
         # Standoff is 8cm over the grasp for now
         # Overwrite standoff pos.z with a really high value so it comes in from above
-        standoff_pos = grasp_pos + np.array([0.0, 0.0, 0.4])
+        standoff_pos = grasp_pos + np.array([0.0, 0.0, 0.6])
```

Comment:
This should probably be an argument to the grasping class instead, so that you can just pass it in? I don't like changing hard-coded values